docs/reference/modules/cluster/remote-clusters-migration.asciidoc

```diff
@@ -0,0 +1,242 @@
+
+. Use the <<cluster-remote-info,remote cluster info API>> to verify that the
+local cluster has successfully connected to the remote cluster:
++
+[source,console]
+----
+GET /_remote/info
+----
+// TEST[skip:TODO]
++
+The API response should indicate that the local cluster has connected to the
+remote cluster:
++
+[source,console-result]
+----
+{
+  "my_remote": {
+    "connected": true, <1>
+    "mode": "proxy",
+    "proxy_address": "my.remote.cluster.com:9443",
+    "server_name": "",
+    "num_proxy_sockets_connected": 0,
+    "max_proxy_socket_connections": 18,
+    "initial_connect_timeout": "30s",
+    "skip_unavailable": false,
+    "cluster_credentials": "::es_redacted::" <2>
+  }
+}
+----
+// TEST[skip:TODO]
+<1> The remote cluster is connected.
+<2> If present, indicates the remote cluster has connected using API key 
+authentication.
+
+[[remote-clusters-migration-resume]]
+==== Resume cross-cluster operations
+
+Resume any persistent tasks that you stopped earlier:
+
+* Use the <<start-transform>> API to start any transforms.
+* Use the <<ml-open-job>> API to open any anomaly detection jobs.
+* Use the <<ccr-post-resume-follow>> API to resume any auto-follow {ccr}.
+* Use the <<ccr-resume-auto-follow-pattern>> API to resume any manual {ccr} or 
+existing indices that were created from the auto-follow pattern.
+
+[[remote-clusters-migration-disable-cert]]
+==== Disable certificate based authentication and authorization
+
+//TODO: Add link to troubleshooting docs when they're published
+NOTE: Only proceed with this step if the migration has been proved successful on
+the local cluster. If the migration is unsuccessful, either find out what the
+problem is and attempt to fix it or <<remote-clusters-migration-rollback,roll
+back>>.
+
+Next, disable the certification based connection and revoke the authorization.
+
+. There is no particular setting to enable or disable a certificate based cross
+cluster connection, because it shares the same transport protocol with the
+intra-cluster node-to-node communication.
++
+One way a remote cluster administrator can stop an existing local cluster from
+connecting, is by changing TLS trust. The exact steps vary, depending on how the
+clusters have been configured. A generic solution is to recreate the CA and
+certificate/key used by the remote transport interface so that any existing
+certificate/key, locally or distributed, is no longer trusted.
++
+Another solution is to apply IP filters to the transport interface, blocking
+traffic from outside the cluster.
+
+. For certificate based cross-cluster operations, the remote cluster manages the
+role definitions. A remote cluster administrator can delete any roles used
+purely for cross-cluster operations. However:
++
+* You can't disable the `superuser` role.
+* API keys bring their privilege definition across clusters. They do not rely on
+named roles on the remote cluster.
```

Comment:
Since this step does `not` really work. We should make it clear that rotating certificate and break TLS trust between the two cluster via the transport interface is the ultimate and safe option to truly stop RCS 1.0 communication between the two clusters. This step is nice to have. But failing to do so won't be a security issue as long as the trust is stopped.